src/PBot/Issues/MoveIssue.cs

```diff
@@ -6,7 +6,7 @@ public class MoveIssue : BotCommand
     {
         public MoveIssue()
             : base("move issue (.*) to (.*)$",
-            "pbot move issue <repository>#<issue number>(or https://github.com/Particular/repository/issues/issuenumber) to <target repository>(or https://github.com/Particular/repository) - Moves an issue from one repository to the other.")
+            "`pbot move issue <repository>#<issue number>(or https://github.com/Particular/repository/issues/issuenumber) to <target repository>(or https://github.com/Particular/repository)` - Moves an issue from one repository to the other. Comments aren't moved, so revising description of new issue is advised.")
```

Comment:
@sergioc I've found that comments are usually moved.

However, it did seem to be failing intermittently on some comments when you were moving issues last week.